### PR TITLE
AB#1054: cyclonedx unpinned assets

### PR DESCRIPTION
**Pull request by Anthony-Mckale**
MAIN
- updated unpinned packages to be included in cyclonedx bom
- changed warning for unpinned package into a vulnerability

OTHER STUFF

- removing safety as a default plugin
- removing dead languages code
- plumbing transitive packages into markdown/html reporter
- removing unused language metadata (unneeded can be determined by file type)

### CODE REVIEW

**Review by mgonzales-rs — CHANGES_REQUESTED** [review]

**Comment by mgonzales-rs** [comment]
cool: 
image

File: eze/plugins/tools/python_cyclonedx.py [hunk 1]
```diff
@@ -121,12 +158,20 @@ async def run_scan(self) -> ScanResult:
...
+            # AB#1054: additional parsing for unpinned assets
+            # add unpinned assets as vulnerabilities
+            # append unpinned assets into cyclonedx
+            unpinned_requirements = self.extract_unpinned_requirements(completed_process_stdout, requirements_file)
+            vulnerabilities_list.extend(unpinned_requirements["vulnerabilities"])
+            components = py_.get(sboms[requirements_file], "components") or []
+            components.extend(unpinned_requirements["cyclonedx_components"])
+            sboms[requirements_file]["components"] = components
 
         for poetry_file in poetry_files:
```
[the remark this answers is not in the document]
Comment (Anthony-Mckale) [reply eze/plugins/tools/python_cyclonedx.py:176]: so did it the otheer way, [warnings, cyclonedx_bom]  to [warnings, cyclonedx_bom, completed_process_stdout], testong on poetry examples

[the remark this answers is not in the document]
Comment (Anthony-Mckale) [reply eze/plugins/tools/python_cyclonedx.py:176]: worked fine for me, if you could check again once i've merged

[the remark this answers is not in the document]
Comment (Anthony-Mckale) [reply eze/plugins/tools/python_cyclonedx.py:176]: tested with eze-goats\goat-python\python-autorun

**Pull request merged**